### PR TITLE
KEYCLOAK-18512: Integrate New Admin Console into Keycloak build

### PR DESCRIPTION
**Pull request by ssilvert**
Ready to go.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2025,4 +2030,4 @@
         </profile>
 
     </profiles>
-</project>
+</project>
```
Comment (jonkoops) [inline pom.xml:2033]: Some rando whitespace change here.

File: pom.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <name>Keycloak</name>
     <description>
         Keycloak SSO
-    </description>
+</description>
```
Comment (jonkoops) [inline pom.xml:31]: Some rando whitespace change here, seems to be misaligned.

File: pom.xml (hunk 1 above)
Comment (jonkoops) [inline pom.xml:2033]: Some rando whitespace change here.

File: services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java [hunk 3]
```diff
@@ -196,6 +191,23 @@ private void setThemes(ServerInfoRepresentation info) {
...
+        
+        boolean filterAccountV2 = (type == Theme.Type.ACCOUNT) && 
+                !Profile.isFeatureEnabled(Profile.Feature.ACCOUNT2);
+        boolean filterAdminV2 = (type == Theme.Type.ADMIN) && 
+                !Profile.isFeatureEnabled(Profile.Feature.ADMIN2);
+        
+        if (filterAccountV2 || filterAdminV2) {
+            filteredNames.remove("keycloak.v2");
+            filteredNames.remove("rh-sso.v2");
+            return filteredNames;
```
Comment (jonkoops) [inline services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java:206]: This return statement seems unnecessary as this value is also returned on the last line of the method.

File: themes/pom.xml [hunk 4]
```diff
@@ -10,7 +9,7 @@
 
     <artifactId>keycloak-themes</artifactId>
     <name>Keycloak Themes</name>
-    <description/>
+    <description />
```
Comment (jonkoops) [inline themes/pom.xml:12]: Unrelated whitespace change.

File: themes/pom.xml [hunk 5]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (jonkoops) [inline themes/pom.xml:2]: Unrelated whitespace change.

File: themes/pom.xml [hunk 6]
```diff
@@ -243,4 +288,4 @@
         </profile>
     </profiles>
 
-</project>
+</project>
```
Comment (jonkoops) [inline themes/pom.xml:291]: Unrelated whitespace change.

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java (hunk 3 above)
Comment (ssilvert) [reply services/src/main/java/org/keycloak/services/resources/admin/info/ServerInfoAdminResource.java]: Fixed

File: pom.xml (hunk 2 above)
Comment (ssilvert) [reply pom.xml]: Fixed this one, which was caused by "format on save".  The rest of the whitespace issues are arguably improvements so I left them alone.

**Review by jonkoops — APPROVED** [review]
LGTM :heavy_check_mark:

**Comment by jonkoops** [comment]
@stianst I think we are good to go here, could you go ahead and give this PR a look?

**Review by stianst — APPROVED** [review]
LGTM and works fine on WildFly, but couldn't get it to work on Quarkus as the keycloak.v2 theme doesn't pop-up. Can be looked into as a follow-up though as it's probably something with Quarkus dist rather than this PR.
